src/lib/y2network/sysconfig/connection_config_readers/wireless.rb

```diff
@@ -17,48 +17,40 @@
 # To contact SUSE LLC about this file by physical or electronic mail, you may
 # find current contact information at www.suse.com.
 
+require "y2network/sysconfig/connection_config_readers/base"
 require "y2network/connection_config/wireless"
```

Comment:
And now with last changes this require is even not needed, isn't it?

Reply:
Yes, it is not needed. I removed that line from `ethernet.rb` but not from here.